<?xml version="1.0"?>
<launch>

    <!--- Sim Time -->
    <param name="/use_sim_time" value="true" />

    <!--- Run Rviz-->
<!--    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find lego_loam)/launch/test.rviz" />-->

    <!--- LeGO-LOAM -->
    <node pkg="lego_loam" type="simpleVO"    name="simpleVO"    output="screen">
        <remap from="/camera/rgb/image_raw" to="/camera/rgb/image_color" />
        <remap from="/camera/depth_registered/image_raw" to="/camera/depth/image" />
    </node>


</launch>
